README.md

```diff
@@ -70,60 +83,73 @@ admission controller too, for that follow the instructions
+
 - `minikube delete`
 
-**NOTE**: this will delete the whole k8s cluster, if you are playing with other things in that same cluster, you might want want to delete each namespace/resource one by one instead.
+**NOTE**: this will delete the whole K8s cluster, if you are playing with other things in that
+same cluster, you might want want to delete each namespace/resource one by one instead.
+
+#### If you installed Harbor with this guide
 
-#### If you installed harbor with this guide
-This will not remove the volumes on harbor side, to do so you'll have to stop harbor:
+This will not remove the volumes on the Harbor side, to do so you'll have to stop Harbor:
 
 - `docker-compose -f .harbor/harbor/docker-compose.yml down -v --remove-orphans`
 
-And you'll need to delete the data directory (sudo is needed due to files created/modified inside the containers):
-- `sudo rm -rf .harbor/harbor/data`
+Then, delete the data directory (sudo is needed due to files created/modified inside the containers):
 
+- `sudo rm -rf .harbor/harbor/data`
 
 ### Notes on setting up Harbor in production
 
-In development, every component of Harbor is deployed via docker-compose. In production, we use an external database. Currently, only PostgreSQL is supported by Harbor. This PostgreSQL db should be set up as a Trove instance with a Cinder volume.
-
-To configure Harbor for use with an external db, you need to uncomment the `external_database` section in `/srv/ops/harbor/harbor.yml` and fill in the necessary information. For the config changes to take effect, you need to run the `/srv/ops/harbor/install.sh` script afterwards. Remember to also remove the db service from Harbor's `docker-compose.yml`.
+In development, every component of Harbor is deployed via docker-compose. In production, we use an external database.
+Currently, only PostgreSQL is supported by Harbor. This PostgreSQL db should be set up as a Trove instance with a Cinder volume.
 
-On toolsbeta, Harbor is deployed on toolsbeta-harborweb-2. The configuration is under `srv/ops/harbor`, and the data under `/srv/harbor`. Puppet code can be found here: https://gerrit.wikimedia.org/r/plugins/gitiles/operations/puppet/+/refs/heads/production/modules/profile/manifests/toolforge/harbor.pp
+To configure Harbor for use with an external db, you need to uncomment the
+`external_database` section in `/srv/ops/harbor/harbor.yml`and fill in the
+necessary information. For the config changes to take effect, you need to run the
+`/srv/ops/harbor/install.sh` script afterwards.
+Remember to also remove the db service from Harbor's `docker-compose.yml`.
 
-**NOTE**: For production, TBD
+On toolsbeta, Harbor is deployed on toolsbeta-harborweb-2. The configuration is under `srv/ops/harbor`, and the data under `/srv/harbor`. Puppet code can be found here: https://gerrit.wikimedia.org/r/plugins/gitiles/operations/puppet/+/refs/heads/production/modules/profile/manifests/toolforge/harbor.pp
```

Comment:
Can we remove the `-2` from the instance name if we only have one instance?

Reply:
I don't know, can an instance be renamed after creation? Let's ask this question on IRC.